crates/external_services/src/aws_kms/decrypt.rs

```diff
@@ -1,6 +1,7 @@
+//! decrypting data using the AWS KMS SDK.
 use common_utils::errors::CustomResult;
 
-use super::*;
+use crate::aws_kms::core::{AwsKmsClient, AwsKmsError, AWS_KMS_CLIENT};
 
 #[async_trait::async_trait]
 /// This trait performs in place decryption of the structure on which this is implemented
```

Comment:
I believe we'll be eventually removing this trait?

Reply:
Yes, will take this up in next PR